### PR TITLE
Add missing migrate types

### PR DESCRIPTION
**Pull request by JordyDeGroeve**

### CODE REVIEW

**Review by tiagobastos — DISMISSED** [review]
Should we add this change to the backlog of changes?

**Review by joaoinacio — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -5207,6 +5208,9 @@ Translates an ID from the deprecated API into a new UUID.
                 + creditNote
                 + subscription
                 + quotation
+                + timeTracking
+                + deliveryNote
```
Comment (mrtus) [inline apiary.apib]: Don't have any api yet for this, so no need to document it yet.

Comment (JordyDeGroeve) [reply apiary.apib]: Indeed. I'll remove it.

**Review by mrtus — DISMISSED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -5262,6 +5267,8 @@ Translates an ID from the deprecated API into a new UUID.
                     + creditNote
                     + subscription
                     + quotation
+                    + timeTracking
+                    + customField
```
Comment (mrtus) [inline apiary.apib:5271]: Q: Should be `customFieldDefinition` I think?

Comment (JordyDeGroeve) [reply apiary.apib:5271]: It is.. 🙈

File: apiary.apib [hunk 3]
```diff
@@ -424,6 +424,7 @@ We list all backwards-compatible additions here. These are currently available i
 
 ### March 2023
 
+- We added `customField`,`timeTracking` and `department` as allowed types on `migrate.id`.
```
Comment (tiagobastos) [inline apiary.apib:427]: Now, the change log doesn't match the added types.

**Review by mrtus — APPROVED** [review]

File: apiary.apib (hunk 3 above)
Comment (JordyDeGroeve) [reply apiary.apib:427]: It does. There was not really a concept like CF definition/value in API v1. Hence it gets transposed from `customField` to `customFieldDefinition` in the migrate call.

**Review by tiagobastos — APPROVED** [review]

**Review by JorenSaeyTL — APPROVED** [review]

**Pull request merged**